pom.xml

```diff
@@ -111,19 +111,19 @@
         <dependency>
             <groupId>org.seleniumhq.selenium</groupId>
             <artifactId>selenium-server</artifactId>
-            <version>3.4.0</version>
+            <version>3.9.1</version>

```

Comment:
Are these updates necessary?
The test should ideally be runnable against JDK7.

Reply:
I can switch the JDK version back to JDK 7, but annoyingly the maven dependency updates are necessary since they were causing failures

Reply:
I don't think that will work - I'm pretty sure the version of selenium etc. we had before are the last ones compatible with JDK7.

Reply:
Hmm that's annoying. I'll have a look at making it work on JDK 7